includes/class-windows-azure-rest-api-client.php

```diff
@@ -985,10 +986,11 @@ public function put_blobs( $container, array $files = array() ) {
 	 * @param string $local_path               Local path.
 	 * @param string $remote_path              Remote path.
 	 * @param bool   $force_direct_file_access Whether to force direct file access.
+	 * @param string $content_type             File content type
 	 *
 	 * @return bool|string|WP_Error Newly put blob URI or WP_Error|false on failure.
 	 */
-	public function put_blob( $container, $local_path, $remote_path, $force_direct_file_access = false ) {
+	public function put_blob( $container, $local_path, $remote_path, $force_direct_file_access = false, $content_type = 'application/octet-stream' ) {
```

Comment:
@hugosolar At what point are we overriding the default of octet-stream with jpeg for instance?

Reply:
@rickalee here we're sending the mime type
https://github.com/10up/windows-azure-storage/blob/2a9e451eb259e40d3c6fa6e7205a2c60af2ade12/includes/class-windows-azure-helper.php#L483
and here
https://github.com/10up/windows-azure-storage/blob/2a9e451eb259e40d3c6fa6e7205a2c60af2ade12/includes/class-windows-azure-helper.php#L417